app/controllers/notes_controller.rb

```diff
@@ -1,7 +1,8 @@
 class NotesController < ApplicationController
 
   def index
-    @note = Note.all
+    @q = Note.ransack(params[:q])
```

Comment:
This variable is not needed in view or anywhere else so just local variable will be enough

```ruby
q = Note.ransack(params[:q])
```

this way we make sure that data is not used in places where it shouldn't be

Reply:
I created the instance variable here because I used it in the note index view to make a searching form so is it still wrong?

Reply:
Ooops, you are right, my bad :)